staging/src/k8s.io/apiextensions-apiserver/test/integration/fixtures/server.go

```diff
@@ -106,3 +111,25 @@ func IntegrationEtcdServers() []string {
 	}
 	return []string{"http://127.0.0.1:2379"}
 }
+
+// NewCRDTestEtcdClient creates an etcd client configured to access rest storage.
+func NewCRDTestEtcdClient(t *testing.T, restOptions *generic.RESTOptions) *clientv3.Client {
```

Comment:
Move to `test/integration/storage` package?

Reply:
Looks like we can drop this and just call https://github.com/kubernetes/kubernetes/blob/7955a2865c34fa49484831b3c9bdbc404ce00cbd/test/integration/utils.go#L72. Applying.

Reply:
W cannot call k/test/integration from here.